user-licence-lifecycle-manager/src/test/java/de/intension/rest/LicenceResourceIT.java

```diff
@@ -124,11 +130,30 @@ private Pair<String, String> getDatabaseEntry() throws Exception {
 
     @Test
     void should_return_404_when_hmac_id_does_not_exist() throws Exception {
-        var request = HttpRequest.newBuilder(URI.create("http://localhost:" + keycloak.getHttpPort() + "/auth/realms/fwu/licences/invalid-hmac-id")).GET().build();
+        var accessToken = getAccessToken();
+        var request = HttpRequest.newBuilder(URI.create("http://localhost:" + keycloak.getHttpPort() + "/auth/realms/fwu/licences/invalid-hmac-id")).GET().header("Authorization", "Bearer " + accessToken).build();
```

Comment:
nice to have: can you update it using `keycloak.getAuthServerUrl()` (also other occurences in THIS class)